testbench/common.py

```diff
@@ -662,8 +664,8 @@ def streamer():
                     yield d[r:chunk_end]
                 if bytes_yield >= limit:
                     database.dequeue_next_instruction(test_id, method)
-                    if conn is not None:
-                        conn.close()
+                    if conn_channel is not None:
```

Comment:
All these variables that gained a `_channel` suffix probably should be called simply `channel`.  They hold a waitress channel (whatever that is).